src/deploy-cromwell-on-azure/scripts/mount.blobfuse

```diff
@@ -39,9 +44,16 @@ else
     write_log "Error mounting $1. Either the account key or the SAS token must be specified." && exit 1
 fi
 
-/usr/bin/blobfuse $1 --tmp-path=$tmp_dir -o nonempty -o attr_timeout=240 -o entry_timeout=240 -o negative_timeout=120 -o allow_other --container-name=$container_name --log-level=LOG_CRIT
-exitcode=$?
-[ $exitcode -eq 0 ] && write_log "Successfully mounted $1" || write_log "Error mounting $1. Exit code: $exitcode" && exit $exitcode
-
+error_message=$(/usr/bin/blobfuse $1 --tmp-path=$tmp_dir -o nonempty -o attr_timeout=240 -o entry_timeout=240 -o negative_timeout=120 -o allow_other --container-name=$container_name --log-level=LOG_CRIT 2>&1 > /dev/null) && fuse_exit_code=0 || fuse_exit_code=$?
 
+if [[ $fuse_exit_code != 0 ]]; then
+    if [[ $account_key != "" ]]; then
+        write_log "Error mounting $1 using the account key. Exit code: $fuse_exit_code. Message: $error_message"
+        exit $fuse_exit_code
```

Comment:
Since the mounts using the account key are those for which mount_containers.sh just obtained the access key, they should work, otherwise something is wrong, so we fail in that case (will halt startup). For SAS tokens, user enters them and they can be wrong, expired etc. so we log a warning and continue (not halting startup).